randovania/games/samus_returns/pickup_database/pickup-database.json

```diff
@@ -307,10 +316,11 @@
             "progression": [
                 "Gravity"
             ],
-            "default_shuffled_count": 1,
+            "default_shuffled_count": 0,
             "default_starting_count": 0,
             "preferred_location_category": "major",
-            "description": "Gravity Suit automatically gives you Varia Suit.\nIt's recommended to use Progressive Suit instead."
+            "description": "Gravity Suit automatically gives you Varia Suit.\nIt's recommended to use Progressive Suit instead.",
+            "original_location": 124
         },
         "Progressive Suit": {
```

Comment:
Heads up for future PR, would be nice to get rid of these, and do progressive stuff like the other games do it.

Reply:
I'm not really sure what you mean by this

Reply:
MSR:
![grafik](https://github.com/randovania/randovania/assets/38186597/6be824da-5b84-4f40-b21e-25fbf9ae691f)
Dread:
![grafik](https://github.com/randovania/randovania/assets/38186597/a5aa31d9-1eaf-431b-8c96-b5dc4db3555a)
![grafik](https://github.com/randovania/randovania/assets/38186597/58882010-c45c-4e72-bfc6-2009268ea6a9)